Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -40,6 +72,13 @@ public static int[] RemoveLast(int[] array)
         public static int[] RemoveFirst(int[] array)
         {
             // ToDo: implement.
+            if (array != null || array.Length > 0)
+            {
+                int[] newArr = new int[array.Length - 1];
+                for (int i = 1; i < array.Length; ++i)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'for' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4940124)